QueryBuilder.Tests/Snowflake/SnowflakeParameterTests.cs

```diff
@@ -0,0 +1,83 @@
+using SqlKata.Compilers;
+using SqlKata.Tests.Infrastructure;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace SqlKata.Tests.Snowflake
+{
+    public class SnowflakeParameterTests : TestSupport
+    {
+        private readonly SnowflakeCompiler compiler;
+
+        public SnowflakeParameterTests()
+        {
+            compiler = Compilers.Get<SnowflakeCompiler>(EngineCodes.Snowflake);
+        }
+
+        [Fact]
+        public void TestDateTimeParameter()
+        {
+            {
+                var dt = new DateTime(1997, 04, 13, 07, 30, 00, DateTimeKind.Utc);
```

Comment:
Feels like we should have a test with `DateTimeKind.Local` since that one is accepted too (or maybe the compiler should throw on `DateTimeKind.Local`, too?)

Reply:
I had wanted to discuss how to handle this with you before merging :)

Reply:
A suggestion would be to mock the timezone so that we know it is e.g. "NewYork"  and then do a test with DateTimeKind.Local, too.

Reply:
Mocking is hard, the official micro-soft fakes shimming library is only available to enterprise users, and open source alternatives do not seem to be very reliable. For now I've opted to reject local time zones for the snowflake compiler, as we're the only ones that will be using that code.